.autoupdate/preupdate

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script is called by our weekly dependency update job in Jenkins, before updating Ruby and other deps
+
+# Without these ENV changes, python gets confused and tries to use latin1
+# encodings which breaks the poetry update below
+export LANG=en_US.UTF-8
+export LC_COLLATE=C.UTF-8
+
+pip3 install --upgrade --requirement requirements.txt > dlme-airflow.txt &&
```

Comment:
i don't see a `requirements.txt` (unlike some of our other airflow/python projects).  no pref between just adding one with `poetry` as the only dep, vs doing a `pip3 install poetry` here (or whatever the right incantation would be).

Reply:
maybe very slight pref for adding `requirements.txt` for consistency? 🤷